app/models/loc_id_base_model.rb

```diff
@@ -43,15 +48,12 @@ def locid=(string)
     end
   end
 
-  def query_locid
-    LocId.where(specific_id: id,
-                specific_type: normalized_class.to_s)
-  end
-
   def normalized_class
     # Sometimes the objects are "Ontology" and sometimes a subclass.
     if [DistributedOntology, SingleOntology].include?(self.class)
       Ontology
+    elsif [Axiom, Theorem].include?(self.class)
```

Comment:
Maybe we could use `if Sentence.descendants.include?(self.class)` at this point.

The same does not work for `Ontology`, at least on my machine: 
```ruby
[1] pry(main)> Ontology.descendants
=> []
[2] pry(main)> Sentence.descendants
=> [Axiom(id: integer, ontology_id: integer, name: text, text: text, range: string, comments_count: integer, created_at: datetime, updated_at: datetime, display_text: text, imported: boolean, type: string, proof_status_id: string, state: string, state_updated_at: datetime, last_error: text, provable: boolean),
 Theorem(id: integer, ontology_id: integer, name: text, text: text, range: string, comments_count: integer, created_at: datetime, updated_at: datetime, display_text: text, imported: boolean, type: string, proof_status_id: string, state: string, state_updated_at: datetime, last_error: text, provable: boolean)]
```
I wonder why this is.